doc/node.1

```diff
@@ -282,6 +282,9 @@ or servers.
 Set default minVersion to 'TLSv1.3'. Use to disable support for TLSv1.2 in
 favour of TLSv1.3, which is more secure.
 .
+.It Fl -trace
+Print the default trace on standard output.
```

Comment:
We generally document only very few V8 options – if you feel that this is important, that’s okay, but can you update the description to explain a bit more on what this does? E.g. the text from `node --v8-options` is `trace function calls`, which doesn’t say much either, but is a bit more explicit than `default trace`.